tests/lmerge_tests.py

```diff
@@ -15,7 +15,8 @@ def run_integration_test(self):
         test_directory = os.path.dirname(os.path.abspath(__file__))
         self.test_data_dir = os.path.join(test_directory, 'test_data', 'lmerge')
         input_file = os.path.join(self.test_data_dir, 'input')
-        expected_result = os.path.join(self.test_data_dir, 'expected.vcf')
+        #expected_result = os.path.join(self.test_data_dir, 'expected.vcf')
```

Comment:
Two more questions:

1. We don't have a test of including the genotypes. Has that been tested at all with the new code?
2. Seems like we're adding a format field to the output now. I assume that doesn't matter, but I think that makes the output an invalid VCF.